pyron-plugin/src/main/scala/com/cloudentity/pyron/plugin/util/value/ValueResolver.scala

```diff
@@ -6,21 +6,36 @@ import io.vertx.core.json.{JsonArray, JsonObject}
 
 import java.util.{List => JavaList, Map => JavaMap}
 import scala.annotation.tailrec
+import scala.collection.JavaConverters._
 import scala.util.Try
 import scala.util.matching.Regex
 
 
 object ValueResolver extends ValueResolver
 trait ValueResolver {
+
```

Comment:
I would move code related to dynamic value (i.e. the one with 'path', 'pattern', 'output'), because it's complex enough (ca 50% of the file is related to this case, but it's only 1 of 5 cases). It got difficult to read a bit.

Reply:
I kind of refrained from moving the code to different files to avoid potential breakage. But I plan to do exactly that, probably in one separate PR.